scripts/py_matter_idl/examples/README.md

```diff
@@ -0,0 +1,19 @@
+## Creating a custom matter_idl generator
```

Comment:
is there a way to unit test the example? That way we see what would get generated and also see if any changes affect outputs.